src/Client/Extensions/HttpClientPushedAuthorizationExtensions.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using IdentityModel.Internal;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace IdentityModel.Client;
+
+/// <summary>
+/// HttpClient extensions for OIDC discovery
+/// </summary>
+public static class HttpClientPushedAuthorizationExtensions
+{
+    /// <summary>
+    /// Sends a pushed authorization request
+    /// </summary>
+    /// <param name="client">The client.</param>
+    /// <param name="request"></param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns></returns>
+    public static Task<PushedAuthorizationResponse> PushAuthorizationAsync(this HttpClient client, PushedAuthorizationRequest request, CancellationToken cancellationToken = default)
+    {
+        var clone = request.Clone();
+    
+        AddOptionalIfNotPresent(clone.Parameters, OidcConstants.AuthorizeRequest.ResponseType, request.ResponseType);
```

Comment:
Thoughts on doing things this way? We have both a request model and its parameters collection, and we're combining all of that together, with the parameters collection taking precedence.